src/utils/logging.cpp

```diff
@@ -1,400 +1,8 @@
-// TODO: Move this to another place and make it #define agnostic
-// Ideally, we would be able to call a one time setup function, perhaps with a ModInfo parameter
-// This would either return or initialize a logger instance for us to use with future calls to "log"
+#include "shared/utils/logging.hpp"
 
-#include "../../shared/utils/logging.hpp"
-#include "modloader/shared/modloader.hpp"
-#include <string_view>
-#include <string>
-#include <memory>
-#include "../../shared/utils/utils-functions.h"
-#include "../../shared/utils/utils.h"
-#include <fstream>
-#include <chrono>
-#include <ctime>
-#include <iomanip>
-#include <sstream>
+static Paper::LoggerContext loggerContext =
```

Comment:
Move this static into the function call (lets try and cut down on the number of global statics we have, they suck).

Also, it's worth noting that we now have no easy way of converting code that debug logs to avoid debug logging (ex, like we normally would via means by checking against `NDEBUG`) We could maybe still do something here where we have `getLogger()` return an empty instance, but that would stop ALL logging, not just debug logging.